chalk-ir/src/debug.rs

```diff
@@ -178,7 +178,9 @@ impl<I: Interner> Debug for TyData<I> {
         match self {
             TyData::BoundVar(db) => write!(fmt, "{:?}", db),
             TyData::Dyn(clauses) => write!(fmt, "{:?}", clauses),
-            TyData::InferenceVar(var) => write!(fmt, "{:?}", var),
+            TyData::InferenceVar(var, TyKind::General) => write!(fmt, "{:?}", var),
+            TyData::InferenceVar(var, TyKind::Integer) => write!(fmt, "{:?}i", var),
+            TyData::InferenceVar(var, TyKind::Float) => write!(fmt, "{:?}f", var),
```

Comment:
Some new notation is added to distinguish integer/float inference variables from normal ones. Does this make sense, or do we not care about adding more notation for this?

Reply:
I think rustc uses a very similar notation, except maybe we write `?0t` for the general case.

Reply:
Well, `_#22t`, something like that :)